rules-reviewed/fuse/sonicesb/sonic-esb.windup.xml

```diff
@@ -47,39 +45,38 @@
             </when>
             <perform>
                  <hint title="Reference to com.sonicsw.xq.XQServiceContext" effort="4" severity="mandatory">
-                     <message><![CDATA[
-
-                             Sonic ESB services leverage the service(XQServiceContext context) method to implement business logic.  When messages are routed to the service, the service(XQServiceContext context) method is executed.
-                             In Camel, this is achieved by routing messages to the Java Bean via the Camel Route's Bean Component syntax.
-
-                             * [Camel Java Bean Component](http://camel.apache.org/bean.html)
-                             * [Camel Binding Annotations](http://camel.apache.org/parameter-binding-annotations.html)
+                     <message>
+                       <![CDATA[
+                       Sonic ESB services leverage the service(XQServiceContext context) method to implement business logic.  When messages are routed to the service, the service(XQServiceContext context) method is executed.
+                       In Camel, this is achieved by routing messages to the Java Bean via the Camel Route's Bean Component syntax.
 
-                             Camel's Java Bean Component can leverage annotations annotations on the method to specify how Camel Message body values are mapped to the method parameters.  Additionally, the @Handler annotation can be leveraged to setup the default Java Bean method.
-
-                             **For example:**
+                       * [Camel Java Bean Component](http://camel.apache.org/bean.html)
+                       * [Camel Binding Annotations](http://camel.apache.org/parameter-binding-annotations.html)
 
-                             ```java
-                             public void service(XQServiceContext ctx) throws XQServiceException \{
-                                 ...
-                             }
-                             ```
+                       Camel's Java Bean Component can leverage annotations annotations on the method to specify how Camel Message body values are mapped to the method parameters.  Additionally, the @Handler annotation can be leveraged to setup the default Java Bean method.
 
-                             **Should become:**
+                       **For example:**
 
-                             ```java
-                             @Handler
-                             public void service(@Header messageHeader, @Body messageBody, Exchange exchange) \{
-                                 ...
-                             }
-                             ```
+                       ```java
+                       public void service(XQServiceContext ctx) throws XQServiceException \{
+                       ...
```

Comment:
I think that we should revisit how the Rewrite construct {...} is matched.
Forcing users to escape every '{' doesn't seem like a good UX to me.